### PR TITLE
Update queue worker version to 0.5.3

### PR DESCRIPTION
**Pull request by viveksyngh**
Signed-off-by: Vivek Singh <vivekkmr45@yahoo.in>

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
Add a commit for armhf please

File: yaml_armhf/queueworker.yml [hunk 1]
```diff
@@ -21,4 +21,8 @@ spec:
           value: "30s"
         - name: faas_function_suffix
           value: ".openfaas-fn"
+        - name: basic_auth
+          value: "false"
+        - name: secret_mount_path
+          value: "/etc/openfaas"
```
Comment (alexellis) [inline yaml_armhf/queueworker.yml]: Don't think this is right?

Comment (viveksyngh) [reply yaml_armhf/queueworker.yml]: should we remove `secret_mount_path`? I am not sure how `secret_mount_path` works for armhf.

Comment (alexellis) [reply yaml_armhf/queueworker.yml]: I'm not sure. @rdimitrov please can you test?

Comment (rdimitrov) [reply yaml_armhf/queueworker.yml]: Sure, I'll give it a go :+1:

Comment (LucasRoesler) [reply yaml_armhf/queueworker.yml]: @rdimitrov any update? Thanks

**Comment by LucasRoesler** [comment]
@viveksyngh it looks like we have a couple conflicts with master

**Pull request closed without merging**